transport/http/config.go

```diff
@@ -81,21 +81,50 @@ func (ts *transportSpec) Spec() config.TransportSpec {
 // 	transports:
 // 	  http:
 // 	    keepAlive: 30s
+//      maxIdleConnsPerHost: 2
+//      connTimeout: 500ms
+//      connBackoff:
+//        exponential:
+//          min: 100ms
+//          base: 100ms
+//          max: 30s
 //
 // All parameters of TransportConfig are optional. This section may be omitted
 // in the transports section.
 type TransportConfig struct {
 	// Specifies the keep-alive period for all HTTP clients. This field is
 	// optional.
-	KeepAlive time.Duration `config:"keepAlive"`
+	KeepAlive           time.Duration  `config:"keepAlive"`
+	MaxIdleConnsPerHost int            `config:"maxIdleConnsPerHost"`
```

Comment:
It’s unclear in the go docs, but we’re threading the var through without change of name. https://golang.org/pkg/net/http/